fvtest/compilertriltest/ILValidatorTest.cpp

```diff
@@ -57,6 +58,69 @@ INSTANTIATE_TEST_CASE_P(ILValidatorTest, IllformedTrees, ::testing::Values(
     "(method return=Int64 (block (lreturn (sconst 1) )))"                   // lreturn incorrect type. 
     ));
 
+class InvalidPlatformVerifier : public TR::IlVerifier
```

Comment:
Why is this verifier needed?

Reply:
@Leonardo2718  I want to test the verifier rule that I added. And I check other examples, it seems that adding a verifier and use it in the validator test is the way that we can test the rule. Or do you think I should use `AllIlVerifier` in `IlVerifierHelpers.hpp` instead?

Reply:
Uhm, I'm not sure which "other examples" you are referring to. None of the other tests in `ILValidatorTest.cpp` use IL verifiers.

The IL verifiers are completely independent of the ILValidator. They're used to check properties of the IL inside a test. For example, you can have a verifier that checks an optimization correctly transformed your trees. So, I don't think you should need one in this case.

Now that I'm reading through the code again, I think you can actually just add your tests to https://github.com/eclipse/omr/blob/c56c5c432457f865dee99b22495080d5339dc7ef/fvtest/compilertriltest/ILValidatorTest.cpp#L42-L58
with the appropriate `#ifdef`, of course.